AssignmentFourMethods/src/assignmentfourmethods/AssignmentFourMethods.java

```diff
@@ -0,0 +1,138 @@
+            
+            if (streetNumber < 100)
+            {
+                System.out.println("Simon says to ten jumping jacks.");
+            }
+            else if (streetNumber > 600)
+            {
+                System.out.println("Go and get yourself a candy bar.");
+            }
+    
+}
+//used if user enters 15
+public static void fifteen(){
+    Scanner keyboard = new Scanner(System.in);
+    System.out.println("Enter your favorite food that you must eat.");
+       int numberOfMeals;     
+       numberOfMeals=keyboard.nextInt();
+            
+        
+            if (numberOfMeals >= 3)
+            {
+                System.out.println("Looks like you are on the right track!");
+            }
+            else if (numberOfMeals < 3) 
+            {        
+                System.out.println("You need to start eating more pizza.");
+            } 
+}
+//if user enters 20
+public static void twenty(){
+    int yourAge;    
+    Scanner keyboard = new Scanner(System.in);
+
+    System.out.println("Please enter your age, be honest.");
+            yourAge=keyboard.nextInt();
+            
+        
+            if (yourAge >= 19)
+            {
+                System.out.println("Today is your lucky day!");
+        }
+            else if(yourAge <= 18)
+            {  
+                System.out.println("You will not be lucky untill you take a"
+                        + " programming class!");
+                
+            }
+}
+//used if user enters 25
+public static void twentyfive(){
+    int yourBirthYear;     
+    Scanner keyboard = new Scanner(System.in);
+
+    System.out.println("Please enter your birth year.");
+            yourBirthYear=keyboard.nextInt();
+            
+            if (yourBirthYear > 1980)
+            {
+                System.out.println("You were born this way, which is lucky!");
+            }
+            else if (yourBirthYear <1900)
+            {
+                System.out.println("Wow you are really old, but stil lucky!");
+            }
+            else if (yourBirthYear > 2015)
+            {
+                System.out.println("Sorry you are not lucky because yoou "
+                        + "weren't born yet.");
+            }
+}
+//if user enters 100
+public static void hundred(){
+                
+            char letterOne='p';
+            char letterTwo='e';
+            char letterThree='r';
+            char letterFour='f';
+            char letterFive='e';
+            char letterSix='c';
+            char letterSeven ='t';
+            int asciiSum;
+       
+            asciiSum= (int)letterOne + (int)letterTwo + (int)letterThree + (int)letterFour + 
+                    (int)letterFive + (int)letterSix + (int)letterSeven; 
+       
+       
+            System.out.println("The word is 'perfect'.");
+            System.out.println("The sum of the word perfect is " + asciiSum + " .");
+}
+//main part of program 
+    public static void main(String[] args) {
```

Comment:
Did you move this to the bottom because or declaration order. If you are familiar with C++ or JavaScript you might be used to not being able to invoke a method until after it has been defined.

In the Java world we place the entry point at the start of the class and all supporting functions are listed below that. Because we have a compile process this is totally fine.

If that is not the reason you did it take it was a grain of salt but the rule still stays the same.